tests/e2e-test.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Copyright 2025 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+echo "starting rekor"
+docker compose up -d --build --wait --wait-timeout 60
+
+cleanup() {
+    docker compose down
+}
+trap cleanup EXIT
+
+echo "running tests"
+go test -v -tags=e2e ./tests/
```

Comment:
non blocker: Since this file is just a few lines, we could put these in a makefile.

Reply:
I opted to just put it in the github workflow